channels/al.m3u

```diff
@@ -49,8 +47,6 @@ http://79.106.48.2/live/rtsh_shqip_ott_p3/playlist.m3u8
 http://79.106.48.2/live/rtsh_sport_ott_p3/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/k6zUFcQ.png" group-title="",RTV Pendimi
 http://tv.rtvpendimi.com:8081/hls/RTVPendimi.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/S0WUoFW.png" group-title="Sport",Supersport 3
-https://sportbar.biz/lba/123/g8/albaniasport3.m3u8
```

Comment:
Link is working right now , but i noticed the sportbar.biz links are not always online only live during live sportevents they stream.

Reply:
We would need to come up with some kind of signature for such channels, otherwise it will fall under the purge every time.